### DISCUSSION TITLE
Reproducibility of the Simulation between RMG 3.2.0 and 3.0.0

### DISCUSSION THREAD

**Discussion by kokiyaita** [ReactionMechanismGenerator/RMG-Py#2625] [Q&A]
Hello! I'm currently doing some tutorials of RMG. I am using both RMG 3.2.0 and RMG 3.0.0 to compare the results between them. I'm curious about how you adjust the parameters of the input files or utilize some techniques to obtain almost same results between the version 3.2.0 and 3.0.0. Can I ask some help?

For instance, I'm currently working on the "methane_steam" located  in examples/rmg/catalysis/methane_steam/input.py. Please see the input file attached for more detail (input_RMG300.py). When I ran the simulation, I got the below result .(result_RMG300.log).
```
The final model core has 44 species and 118 reactions
The final model edge has 428 species and 1134 reactions
```

Then, I also ran the same calculation on RMG 3.2.0 but I had to change some of the database as follows as the file names of the database have been changed.  

```
For Surface/Deutschmann_Ni,
RMG 3.0.0: Surface/Deutschmann_Ni
RMG 3.2.0: Surface/Methane/Deutschmann_Ni
For SurfaceThermoNi,
RMG 3.0.0: surfaceThermoNi
RMG 3.2.0: surfaceThermoNi111
For SurfaceThermoPt,
RMG 3.0.0: surfaceThermoPt
RMG 3.2.0: surfaceThermoPt111
```
I stopped the simulation in 20 minutes because the previous simulation was done in 8 minutes and the number of the core reactions and species were much less.
```
After model enlargement:
    The model core has 65 species and 629 reactions
    The model edge has 1565 species and 4748 reactions
```

Thank you very much for working on this issue!
[RMG_methane_steam_git.zip](https://github.com/ReactionMechanismGenerator/RMG-Py/files/14505528/RMG_methane_steam_git.zip)

**Comment by JacksonBurns** [comment]
@bjkreitz and/or @ChrisBNEU thoughts on this?
